.scripts/test-coverage.sh

```
@@ -0,0 +1,55 @@
+}
+
+show_cover_report() {
+    go tool cover -${1}="$profile"
+}
+
+push_to_coveralls() {
+    echo "Pushing coverage statistics to coveralls.io"
+    goveralls -coverprofile="$profile"
+}
+
+generate_cover_data $(go list ./...)
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)